scapy/layers/dns.py

```diff
@@ -235,17 +235,16 @@ def i2h(self, pkt, x):
 class DNS(Packet):
     name = "DNS"
     fields_desc = [ ShortField("id", 0),
-                    BitField("qr", 0, 1),
-                    BitEnumField("opcode", 0, 4, {0:"QUERY",1:"IQUERY",2:"STATUS"}),
-                    BitField("aa", 0, 1),
-                    BitField("tc", 0, 1),
-                    BitField("rd", 0, 1),
-                    BitField("ra", 0, 1),
-                    BitField("z", 0, 1),
+                    BitEnumField("response", 0, 1, {0: "Query", 1: "Response"}),
```

Comment:
According to RFC1035, this field is called "qr". Therefore, I don't think that we must change it.